sourmash/lca/lca_db.py

```diff
@@ -4,27 +4,11 @@
 import json
 import gzip
 from collections import OrderedDict, defaultdict, Counter
-import functools
 
-import sourmash
 from sourmash._minhash import get_max_hash_for_scaled
-from sourmash.logging import notify, error, debug
+from sourmash.logging import notify, debug
```

Comment:
it looks to me like you're cleaning up unnecessary imports here, yes? that's great! but it's a little confusing to do that in an unrelated PR and trips up code review - I was like, why are they changing lca_db!?

Reply:
oh, I see, you had to touch this file to move `cached_property` to utils. never mind!